### ISSUE TITLE
Unable to load validation set correctly

### ISSUE THREAD

**Issue by xinyanghuang7** [cliffbb/OEM-Fewshot-Challenge#2]
When I use the code you provided, I get an error when running test.sh.

I strictly follow your environment configuration and usage process, but there are still some problems.

The error is reported as follows:
![image](https://github.com/cliffbb/OEM-Fewshot-Challenge/assets/75032525/d2f8b2c8-a90a-42ae-a5d2-1716087126d1)

I think it might be because of this problem:

**[ WARN:0@50.182] global loadsave.cpp:248 findDecoder imread_('/home/user/huangxinyang/CVPR24workshop/valset (copy)/labels/lohur_7.tif'): can't open/read file: check file path /integrity**

It may be because of the above reasons that the verification set cannot be read correctly.

Please can you provide me with some help.

**Comment by xinyanghuang7** [comment]
And even though I submitted my documents according to the following results, the evaluation system still cannot give me a score:

![image](https://github.com/cliffbb/OEM-Fewshot-Challenge/assets/75032525/f80ba833-019d-4344-a9de-8562ab7626e5)

**Comment by xinyanghuang7** [comment]
And I am unable to display the results correctly when submitting documents according to the specifications:

![image](https://github.com/cliffbb/OEM-Fewshot-Challenge/assets/75032525/0b8e8de8-ab4d-4a4b-aa58-587c1a92a45f)
![image](https://github.com/cliffbb/OEM-Fewshot-Challenge/assets/75032525/173c5999-71aa-49ea-9db9-9b3dda5f263f)

**Comment by cliffbb** [comment]
Hello Xinyanghuang,
Sorry for the inconvenience. I'm checking it. I'll get back to you very soon.
Best regards,
Cliff


And I am unable to display the results correctly when submitting documents according to the specifications:

image.png (view on web)
image.png (view on web)

**Comment by xinyanghuang7** [comment]
I think the model should be updated on the new classes in the training set and then tested on the images in the validation set.

But I don’t know which images in the training set contain new classes and how to update the new classes in the training set.

Thank you for your timely reply, I hope you can answer my questions!

Thank you!

**Comment by cliffbb** [comment]
@xinyanghuang7 

`**[ WARN:0@50.182] global loadsave.cpp:248 findDecoder imread_('/home/user/huangxinyang/CVPR24workshop/valset (copy)/labels/lohur_7.tif'): can't open/read file: check file path /integrity**`

The issue was the data loader was looking for the labels of the `query_set` which you're not given. You're given the `support_set` which contains 20 images and their labels. The `query_set` contains only the images without labels. And you're to you the `support_set` to predict the images in the `query_set`. 
The bug has been fixed. So you can clone the repos and try it again.
Thank you.

```
And I am unable to display the results correctly when submitting documents according to the specifications:
image.png (view on web)
image.png (view on web)
```
You should directly compress the ".png" files and not the folder that contains the files. I think this is a Codalab issue. So the ".zip" when you unzip it will not contain a folder. Thank you

**Comment by xinyanghuang7** [comment]
Thank you very much for your timely reply and correction！
